talk/objectorientation/constructors.tex

```diff
@@ -64,12 +64,12 @@
     struct MyFirstClass {
       int a;
       MyFirstClass();
-      MyFirstClass(int a);
+      explicit MyFirstClass(int a);
     };
     struct MySecondClass : MyFirstClass {
       int b;
       MySecondClass();
-      MySecondClass(int b);
+      explicit MySecondClass(int b);
       MySecondClass(int a, int b);
```

Comment:
I would not introduce `explicit` here, since this is a slide about inheritance.

Reply:
And in general, to answer the original question of @chavid I would avoid using explicit on slides where we want to concentrate on some other feature so that we do not introduce too much noise. But we should probably add it in a number of places in our exercises and solutions !

Reply:
@bernhardmgruber My bad. I first put them in the slides coming after the `explicit`ones, then changed my mind, and I apparently forget a few ones. I should have checked the diff.

Reply:
@bernhardmgruber My bad. I should have checked the diff.